app/routers/frontend.py

```diff
@@ -158,3 +177,30 @@ async def account(
         "aggregates": aggregates,
     }
     return templates.TemplateResponse("account_page.html", context=context)
+
+
+
+
+
+#Route to get the uploaded audio and transcriptions
+@router.get(
+    "/transcriptions", 
+    response_model=List[AudioTranscriptionBase],
+)
+async def get_audio_transcriptions(
+    current_user=Depends(get_current_user), 
+    db: Session = Depends(get_db)
+):
+    transcriptions = db.query(AudioTranscription).filter(
+        AudioTranscription.username == current_user.username
+    ).all()
+
+    if not transcriptions:
+        raise HTTPException(status_code=404, detail="No transcriptions found")
```

Comment:
@katg999 `HTTPException` call looks like is not defined or imported anywhere

Reply:
Rectified by specifying HTTPException from fastapi